### PR TITLE
Add ctype to the ComposeComplete

### LINKED ISSUE #3297: Add ctype to the ComposeComplete schema
Currently the regindexer code that maintains an index of Flatpaks and containers in the Fedora registry looks at the 'ctype' field of a ~mashtask~compose.complete message to know whether the index needs to be rebuilt:

``` python
   match_topic = conf.daemon.topic_prefix + '.' + conf.daemon.environment + '.bodhi.masktask.complete'
   for name, endpoint, topic, raw_msg in fedmsg.tail_messages(topic=match_topic):
        msg = raw_msg['msg']
        if msg.get('ctype') in ('container', 'flatpak'):
            indexer_thread.reindex('Bodhi mash, ctype="%s"' % msg['ctype'])
```

The ctype field is not part of the ComposeCompleteV1 schema - it would be great if it could be added, since I'd like to continue using it when switching to fedora-messaging.

(@bowlofeggs suggested possibly extending the topic - I suppose like `compose.complete.flatpak` - I don't see a reason to go that fancy - there are few compose.complete messages.)

### PR DESCRIPTION
**Pull request by Zlopez**
Fixes #3297

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: bodhi/messages/schemas/compose.py [hunk 1]
```diff
@@ -44,6 +44,10 @@ class ComposeCompleteV1(BodhiMessage):
             'success': {
                 'type': 'boolean',
                 'description': 'true if the compose was successful, false otherwise.'
+            },
+            'ctype': {
+                'type': 'string',
+                'description': 'Type of the compose'
```
Comment (nphilipp) [inline bodhi/messages/schemas/compose.py]: I'd add a period to the end, for consistency (see above descriptions).

**Review by nphilipp — APPROVED** [review]
LGTM, thanks!

**Review by nphilipp — CHANGES_REQUESTED** [review]
Gah, forgot: please squash the commits into one.

File: bodhi/messages/schemas/compose.py (hunk 1 above)
Comment (Zlopez) [reply bodhi/messages/schemas/compose.py]: Fixed

**Comment by Zlopez** [comment]
Squashed commits

**Review by nphilipp — APPROVED** [review]
Great, thanks!

**Pull request merged**